recipes/jupyterlab-translate/build.sh

```diff
@@ -0,0 +1,2 @@
+$PYTHON -m pip install . -vv
+npm install gettext-extract
```

Comment:
If this is jlab ext, please see https://conda-forge.org/docs/maintainer/knowledge_base.html#jupyterlab-extensions

Reply:
Nope this is not a JLab Extension, but it needs a package available on NPM

Reply:
But I could follow that docs, if it still applies?

Reply:
So those only work if the npm package will get picked up by Jlab on the user's machine.  I am not sure how this project works, but I think the npm install here will only install place it into conda-build's working directory, not the PREFIX. So the gettext-extract won't make it into the package

Reply:
Right. What is the right way to bundle an npm package?

Reply:
I think you do a global install

Reply:
Done 🤷